src/inapp/inapp.ts

```diff
@@ -457,12 +457,17 @@ export function getInAppMessages(
             Promise.all(trackRequests).catch((e) => e);
           }
 
+          const ua = navigator.userAgent;
+          const isSafari =
```

Comment:
Do we need to check that the browser is not safari? Can a browser give back safari and chrome or firefox as the user agent?

Reply:
From my testing, I found that in Chrome the user agent looks like this `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36` so then Chrome would be mistakenly thought to be Safari.

Reply:
I just checked Firefox and found that its user agent looks like this `Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:105.0) Gecko/20100101 Firefox/105.0` so it seems that at least I don't need to check that it's not firefox

Reply:
Also checked on Microsoft Edge, whose user agent includes "Chrome" so the Safari check covers that.